frontend/src/metabase/questions/containers/EntityList.jsx

```diff
@@ -175,8 +175,10 @@ export default class EntityList extends Component {
 
         const section = this.getSection();
 
+        const hasEntitiesInPlainState = !(section.section === "all" && entityIds.length === 0);
```

Comment:
I think flipping this would be a little clearer. `entityIds.length > 0 || section.section !== "all"`